fluentnamer/readme.md

```diff
@@ -62,5 +62,4 @@ client-logger: true
 generate-client-interfaces: true
 required-parameter-client-methods: true
 client-flattened-annotation-target: none
-use-input-stream-for-binary: true
```

Comment:
Remove the backward-compatible flag.

If this caused breaking changes to GA lib, we can always add it to swagger readme, for that particular lib.